interactive-tests/interactives_interactions_targetGameDist.rb

```diff
@@ -1,7 +1,6 @@
 # Run basic test first.
 load 'interactive-tests/default.rb', true
-$test.click_element('#main-help-icon')
-$test.save_screenshot
+
```

Comment:
Why was this removed? It doesn't seem to be related to About dialog. It checks help system actually.